### PR TITLE
Replace codecov.io by codecov

### PR DESCRIPTION
**Pull request by GuillaumeAmat**
The first one carries some vulnerabilities and as we can see in the different repos, it is not soon to be fixed...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,11 +3,11 @@ language: node_js
...
 - 8
 before_install:
-- npm install codecov.io coveralls
+- npm install codecov coveralls
 - npm install nwb
 after_success:
-- cat ./coverage/lcov.info | ./node_modules/codecov.io/bin/codecov.io.js
-- cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
+- cat ./coverage/lcov.info | ./node_modules/.bin/codecov --pipe
+GuillaumeAmat-patch-1- cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```
Comment (embiem) [inline .travis.yml]: Could you explain this change to me please? I don't understand why there's a reference to your patch branch in there

Comment (GuillaumeAmat) [reply .travis.yml]: Woops... No reason at all!... Sorry :/

I'm fixing it.

Comment (GuillaumeAmat) [reply .travis.yml]: Done @embiem

**Comment by GuillaumeAmat** [comment]
You're welcome! Thank you for your useful library 👌

**Comment by GuillaumeAmat** [comment]
@embiem do you plan to release a new version in the next days or weeks?

**Comment by GuillaumeAmat** [comment]
@embiem ?

**Pull request merged**